pymc3/distributions/continuous.py

```diff
@@ -1350,6 +1350,15 @@ def logp(self, value):
         return bound(logp, value >= 0, value <= 1, a > 0, b > 0)
 
 
+class PyMC3ExponentialRV(ExponentialRV):
```

Comment:
What do we need this for?

Reply:
Because the Aesara ExponentialOp is parametrized in terms of terms of mu, whereas our logp is parametrized in terms of lam. 
https://github.com/pymc-devs/aesara/tree/master/aesara/tensor/random/basic.py#L93

Reply:
I am not defending this approach. I think we still need to figure out how to cleanly link between different parametrizations of logp and random methods. This is just to get it to a point where it is even passing the tests.